### DISCUSSION TITLE
SQL Server Connection

### DISCUSSION THREAD

**Discussion by justinfullers** [pimcore/pimcore#13827] [Q&A / Support]
I'm wondering if there are any available connectors for Pimcore to SQL server?

**Comment by mattamon** [comment]
Hey @justinfullers,

there is tray.io -> https://tray.io/connectors/microsoft-sql-server-pimcore-integrations, I am not sure if it suits your needs.
What are you trying to achieve with an SQL Connector?

**Reply by justinfullers** [reply]
Thanks I will take a look.
